src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/resettlementassessmentstrategies/AbstractResettlementAssessmentStrategy.kt

```diff
@@ -178,6 +183,8 @@ abstract class AbstractResettlementAssessmentStrategy<T, Q>(
 
   abstract fun getPageList(): List<ResettlementAssessmentNode>
 
+  abstract fun getQuestionList(): List<IResettlementAssessmentQuestion>
```

Comment:
Not sure if this is much better but if you wanted to avoid having to override this function in every pathway you could change it to be something like
```
fun getQuestionList(): List<IResettlementAssessmentQuestion> {
  return questionClass.java.enumConstants.asList()
}
```